src/test/java/gov/usgs/wma/waterdata/JsonType.java

```diff
@@ -19,11 +19,7 @@ public JsonType() {
 
 	@Override
 	public Object typeCast(Object value) throws TypeCastException {
-		if (null == value) {
-			return null;
-		} else {
-			return value.toString();
-		}
+		return value.toString();
```

Comment:
@orosz-usgs this change is one I had not intended to get into this PR, but it does follow the same pattern as the other repos.  I can remove it if you want.